tests/parse_metadata.py

```diff
@@ -31,6 +31,8 @@
 
 
 _STUBTEST_PLATFORM_MAPPING: Final = {"linux": "apt_dependencies", "darwin": "brew_dependencies", "win32": "choco_dependencies"}
+QUERY_URL_ALLOWLIST = {"sourceware.org"}
+"""Some older websites have a bad pattern of using query params for navigation"""
```

Comment:
```suggestion
# Some older websites have a bad pattern of using query params for navigation
QUERY_URL_ALLOWLIST = {"sourceware.org"}
```

Reply:
I don't mind either way, but I liked the docstring showing up in IDEs ^^
![image](https://github.com/python/typeshed/assets/1350584/606156ec-4ddc-4a4f-a7b0-8c67c92e2e62)

Reply:
To me it just looks pretty odd doing something I thought was Sphinx-specific when we don't autogenerate any docs from this file. I'd much prefer the comment here, I'm afraid :)